.github/workflows/release_latest_version.yml

```diff
@@ -11,11 +11,11 @@ jobs:
   build:
     runs-on: ubuntu-latest
     env:
-      MODULES: api ui
+      MODULES: api ui ops
```

Comment:
Does making the `ops` module create and activate the python environment?

Reply:
Yes! It's being handled by makerules here: https://github.com/broadinstitute/wfl/blob/6d1e2472f43ddf2baea932de0fcb8dcd4c7064dc/makerules/Python.mk#L24

Reply:
Oh I see!